python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1672,6 +1672,13 @@ def _should_backfill_atomic_asset_subset_unit(
         )
         entity_subset_to_filter = entity_subset_to_filter.compute_difference(requested_partitions)
 
+    any_parent_being_requested_this_tick_subset = any(
```

Comment:
nit: odd name here -- should it just be `any_parent_being_requested_this_tick` given that it's a boolean?